snapshot/bundlestore/store.go

```diff
@@ -0,0 +1,68 @@
+package bundlestore
+
+import (
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/scootdev/scoot/os/temp"
+)
+
+type StoreRead interface {
```

Comment:
Comment exported Members? (Sorry! I know I'm the slacker who pooped out this uncommented code)